Package.swift

```diff
@@ -651,8 +648,8 @@ let package = Package(
         "FirebaseCore",
         "leveldb",
         .product(name: "nanopb", package: "nanopb"),
-        .product(name: "abseil", package: "abseil"),
-        .product(name: "gRPC-cpp", package: "gRPC"),
+        .product(name: "abseil", package: "abseil-cpp-SwiftPM"),
+        .product(name: "gRPC-cpp", package: "grpc-ios"),
```

Comment:
Note: these name changes are due to the package names deduced from the URL (instead of explicitly specifying names in dependency declarations).